### PR TITLE
 BaSP-TG: Task.

### PR DESCRIPTION
**Pull request by RicardoDosko**
This PR contains the following functions for the Task section:
Get all assigned tasks in a list
Get by ID, to get a task by its unique identifier.
Add a new task to the task list.
Delete a task from the list.
Update the task list.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
 // use "import" to import libraries
-import express from 'express';
+const express = require('express');
 
 // use "require" to import JSON files
 const admins = require('./data/admins.json');
+const userRouter = require('./resources/tasks');
```
Comment (ArianaMaldonado) [inline src/index.js:7]: Delete all blank spaces.

File: src/resources/tasks.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const fs = require('fs');
+
```
Comment (ArianaMaldonado) [inline src/resources/tasks.js:3]: Delete all blank spaces.

File: src/index.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
 const app = express();
 const port = process.env.PORT || 3000;
-
+app.use(express.json());
+app.use('/tasks', userRouter);
```
Comment (ArianaMaldonado) [inline src/index.js:11]: This line should be below the port listener.

File: package.json [hunk 4]
```diff
@@ -1,40 +1,40 @@
...
+    "devDependencies": {
+        "@babel/core": "^7.17.9",
+        "@babel/node": "^7.16.8",
+        "@babel/preset-env": "^7.16.11",
+        "eslint": "^8.14.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-plugin-import": "^2.26.0",
+        "husky": "^7.0.4"
+    }
+}
```
Comment (cocolobos) [inline package.json:40]: be careful with the spaces

File: src/data/tasks.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    {"id":"3","description":"Stronghold"},
+    {"id":"4","description":"Keylex"},
+    {"id":"5","description":"Aerified"},
+    {"id":"6","description":"Sonsing"},
+    {"id":"7","description":"Konklux"},
+    {"id":"8","description":"Alphazap"},
+    {"id":"9","description":"Solarbreeze"},
+    {"id":"10","description":"Stringtough"},
+    {"id":"11","description":"Stringtough"},
+    {"id":"12","description":"Biodex"}]
```
Comment (cocolobos) [inline src/data/tasks.json:12]: JSON file needs to be format

File: src/index.js [hunk 6]
```diff
@@ -1,22 +1,22 @@
 // use "import" to import libraries
-import express from 'express';
+const express = require('express');
```
Comment (cocolobos) [inline src/index.js:2]: do we need this change ?

File: src/resources/tasks.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+            res.status(404).send(err);
+        } else {
+            res.status(201).json(tasksjson);
+        }
+    });
+});
+
+
+router.put('/:id', (req, res) => {
+    const Found = taskData.some((tsAdmin) => tsAdmin.id === req.params.id);
```
Comment (cocolobos) [inline src/resources/tasks.js:48]: be careful with the name, should be more specific and using lowerCase

File: src/index.js (hunk 6 above)
Comment (RicardoDosko) [reply src/index.js:2]: no, it is not necessary, it fulfills the same function

**Review by cocolobos — APPROVED** [review]

**Pull request merged**